### ISSUE TITLE
Unable to find SyntaxHighlighter

### ISSUE THREAD

**Issue by chandru9279** [bower/search#63]
I am however able to install it with
```
bower install SyntaxHighlighter --production --save
```

I tried searching with string "SyntaxHighlighter", "syntax"

**Comment by BenMann** [comment]
Just some thoughts:
the repo moved around quite a bit to https://github.com/syntaxhighlighter/syntaxhighlighter, but still resolves correctly when isntalling. The name specified in the [bower.json](https://github.com/syntaxhighlighter/syntaxhighlighter/blob/master/bower.json) is partly uppercase, which does not comply with [naming conventions](https://github.com/bower/spec/blob/master/json.md#name). Thuns it might not show in search!

**Comment by oosidat** [comment]
Do the bower registry (via command-line) and the bower.io/search refer to the same source of data? I believe that the search app looks at the data at https://bower-component-list.herokuapp.com/ (where SyntaxHighlighter doesn't appear to be present).

**Comment by BenMann** [comment]
Exactly.. and that's (I think) because the search-server or client either checks for naming conventions or just casts the package-name to lowercase by default. This is pretty inconsistent and needs fixing on a larger scale => standardization for handling package names!

Can be found in registry and thus be installed:
`curl http://bower.herokuapp.com/packages/SyntaxHighlighter`

Doesn't appear in aggregated list for search: 
`https://bower-component-list.herokuapp.com/`
